### PR TITLE
SONAR-20383 Fix flaky issue on docker buildx

### PR DESCRIPTION
**Pull request by carminevassallo**

### CODE REVIEW

**Comment by steve-marion-sonarsource** [comment]
The commit should be marked with DO NOT MERGE or WIP in it so that there is no way it can be merged.

**Comment by carminevassallo** [comment]
It's not ready for review yet. I'll request your review, @steve-marion-sonarsource when ready.

File: .cirrus/multi-arch-build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export DOCKER_BUILDKIT=1
```
Comment (jCOTINEAU) [inline .cirrus/multi-arch-build.sh:3]: We can reduce a bit the cognitive complexity by doing 

```
export DOCKER_BUILDKIT=1
for i in $(seq 1 3); do
  if docker buildx build --platform linux/amd64,linux/arm64 --tag "$1:$2" --push $3; then
    echo "[Success]: Buildx, attempt ${i}"
    exit 0
  fi
  sleep 3
done
echo "[Error]: Context Deadline Exceeded - Buildx"
exit 1
```

wfyt ?

**Review by jCOTINEAU — APPROVED** [review]
lgtm !

Added a small comment about reducing the complexity of the script, not preventing from merging in my opinion

File: .cirrus/multi-arch-build.sh (hunk 1 above)
Comment (carminevassallo) [reply .cirrus/multi-arch-build.sh:3]: *very* good suggestion! Gonna incorporate it now

**Pull request merged**